### PR TITLE
Add provider for Lithuanian municipalities

### PR DESCRIPTION
**Pull request by norkunas**
This PR:

- [x] Adds a new feature ...
- [x] Covered by tests
- [ ] Fixes #nnnn

### CODE REVIEW

File: src/Faker/Provider/lt_LT/Municipality.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Faker\Provider\lt_LT;
+
+use Faker\Provider\Base;
+
+class Municipality extends Base
```
Comment (IonBazan) [inline src/Faker/Provider/lt_LT/Municipality.php:7]: This should be in `\Faker\Provider\lt_LT\Address`

File: src/Faker/Provider/lt_LT/Municipality.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
...ė", "Klaipėdos rajono savivaldybė", "Kretingos rajono savivaldybė", "Kupiškio rajono savivaldybė", "Lazdijų rajono savivaldybė", "Marijampolės savivaldybė", "Mažeikių rajono savivaldybė", "Molėtų rajono savivaldybė", "Neringos savivaldybė", "Pagėgių savivaldybė", "Pakruojo rajono savivaldybė", "Palangos miesto savivaldybė", "Panevėžio miesto savivaldybė", "Panevėžio rajono savivaldybė", "Pasvalio rajono savivaldybė", "Plungės rajono savivaldybė", "Prienų rajono savivaldybė", "Radviliškio rajono savivaldybė", "Raseinių rajono savivaldybė", "Rietavo savivaldybė", "Rokiškio rajono savivaldybė", "Skuodo rajono savivaldybė", "Šakių rajono savivaldybė", "Šalčininkų rajono savivaldybė", "Šiaulių miesto savivaldybė", "Šiaulių rajono savivaldybė", "Šilalės rajono savivaldybė", "Šilutės rajono savivaldybė", "Širvintų rajono savivaldybė", "Švenčionių rajono savivaldybė", "Tauragės rajono savivaldybė", "Telšių rajono savivaldybė", "Trakų rajono savivaldybė", "Ukmergės rajono savivaldybė", "Utenos rajono savivaldybė", "Varėnos rajono savivaldybė", "Vilkaviškio rajono savivaldybė", "Vilniaus miesto savivaldybė", "Vilniaus rajono savivaldybė", "Visagino savivaldybė", "Zarasų rajono savivaldybė"];
```
Comment (IonBazan) [inline src/Faker/Provider/lt_LT/Municipality.php:13]: Please use single quotes `'` and split this array into multiple lines.

File: test/Faker/Provider/lt_LT/AddressTest.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    {
+        $faker = new Generator();
+        $faker->addProvider(new Address($faker));
+        $this->faker = $faker;
+    }
+
+    public function testMunicipality()
+    {
+        $result = $this->faker->municipality();
+        $this->assertStringEndsWith('savivaldybė', $result);
```
Comment (Nyholm) [inline test/Faker/Provider/lt_LT/AddressTest.php:24]: That is pretty convenient =)

**Review by Nyholm — APPROVED** [review]
Thank you

File: test/Faker/Provider/lt_LT/AddressTest.php (hunk 3 above)
Comment (norkunas) [reply test/Faker/Provider/lt_LT/AddressTest.php:24]: :smile: check in your Sweden provider, i guess it's probably this way better to expect :)

**Review by pimjansen — APPROVED** [review]

**Pull request merged**